### PR TITLE
Add Gcolor3

### PR DESCRIPTION
**Pull request by Hjdskes**
[Gcolor3](https://hjdskes.nl/projects/gcolor3) is my color picking application for the GNOME desktop environment.

### CODE REVIEW

File: nl.hjdskes.gcolor3.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+    "app-id": "nl.hjdskes.gcolor3",
+
```
Comment (TingPing) [inline nl.hjdskes.gcolor3.json]: Just for consistency lets remove the extra whitespace.

File: nl.hjdskes.gcolor3.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
+    "sdk": "org.gnome.Sdk",
+
+    "command": "gcolor3",
+
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
```
Comment (TingPing) [inline nl.hjdskes.gcolor3.json]: Since this doesn't actually work on Wayland perhaps just remove this?

File: nl.hjdskes.gcolor3.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+        "--filesystem=xdg-config/gcolor3:rw"
```
Comment (TingPing) [inline nl.hjdskes.gcolor3.json]: Remove this. You already get your own private config dir.

File: nl.hjdskes.gcolor3.json [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        "/include",
+        "/lib/pkgconfig",
+        "/share/man"
+    ],
+    
+    "modules": [
+        {
+            "name": "gcolor3",
+            "buildsystem": "meson",
+            "builddir": true,
```
Comment (TingPing) [inline nl.hjdskes.gcolor3.json]: Can remove this, it is forced on for meson.

**Comment by TingPing** [comment]
:heavy_check_mark: Upstream appdata file looks good

File: nl.hjdskes.gcolor3.json (hunk 2 above)
Comment (Hjdskes) [reply nl.hjdskes.gcolor3.json]: What do I need that for? It's working fine for me in Wayland at the moment without dconf access.

**Comment by Hjdskes** [comment]
Thanks, commit ammended again.

File: nl.hjdskes.gcolor3.json (hunk 2 above)
Comment (TingPing) [reply nl.hjdskes.gcolor3.json]: GTK on Wayland uses DConf.

File: nl.hjdskes.gcolor3.json [hunk 5]
```diff
@@ -0,0 +1,35 @@
+{
+    "app-id": "nl.hjdskes.gcolor3",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
+    "sdk": "org.gnome.Sdk",
+    "command": "gcolor3",
+
```
Comment (TingPing) [inline nl.hjdskes.gcolor3.json]: There seems to be a misunderstanding here. When I said remove the white space I meant all of these empty lines you have.

Comment (Hjdskes) [reply nl.hjdskes.gcolor3.json]: Ah, misunderstanding indeed.

**Comment by Hjdskes** [comment]
All whitespace removed and DConf added.

**Comment by TingPing** [comment]
bot, build nl.hjdskes.gcolor3

**Comment by TingPing** [comment]
Repository has been created: https://github.com/flathub/nl.hjdskes.gcolor3

**Comment by Hjdskes** [comment]
Thanks!

**Pull request closed without merging**